### PR TITLE
chore: fix setup and start scripts

### PR DESCRIPTION
**Pull request by obdulia-losantos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,28 +167,31 @@ The tests clean up will **DELETE ALL MAPPINGS!!!**
...
 
 ```bash
-docker-compose up --build    # this will update the containers if needed
+./scripts/docker_start.sh [options] [app]
 ```
 
-or
+Arguments:
+  `--force` | `-f`  will kill running containers
+  `--killl` | `-k`  will kill running containers
```
Comment (admbtlr) [inline README.md]: typo?

File: scripts/docker_start.sh [hunk 2]
```diff
@@ -25,22 +25,23 @@
 #
 # arguments:
 #   --force | -f  will kill running containers
+#   --killl | -k  will kill running containers
```
Comment (admbtlr) [inline scripts/docker_start.sh]: typo

File: README.md (hunk 1 above)
Comment (admbtlr) [reply README.md]: and what's the difference between `-f` and `-k`?

File: README.md [hunk 3]
```diff
@@ -167,28 +167,31 @@ The tests clean up will **DELETE ALL MAPPINGS!!!**
...
+  `--build` | `-b`  will build containers before starting them
 
-```bash
-./scripts/docker_start.sh    # starts all
-```
+  `app`
+    Expected values: `kernel`, `odk`, `ui`, `couchdb-sync` or `sync`.
+    Another value will start all containers.
```
Comment (admbtlr) [inline README.md]: Actually "Any other...." is slightly more natural here than "Another..."

**Review by admbtlr — CHANGES_REQUESTED** [review]

**Comment by admbtlr** [comment]
Reading the README, I'm a bit confused: should I be using `./scripts/docker_start.sh` rather than `docker-compose up`? Both are mentioned, and it's not immediately clear to me why...

**Comment by obdulia-losantos** [comment]
`docker_start.sh` takes care of the order in which the containers must be started to work smoothly and without health errors, missing resources...

`docker-compose` starts the containers in the same order they appear in the `docker-compose.yml` file.

We already know that this is not enough :sob:

**Comment by admbtlr** [comment]
I guess this is nothing to do with this PR, but when I run it I get 

```couchdb-sync-rq_1  | psycopg2.OperationalError: FATAL:  database "couchdb-sync" does not exist```

**Comment by admbtlr** [comment]
Also 
image

**Comment by obdulia-losantos** [comment]
As expected if you are not running gather :sweat_smile:

**Comment by obdulia-losantos** [comment]
I'll check the `couchdb-sync-rq` error :see_no_evil:

**Comment by admbtlr** [comment]
Ah, true, good point

**Comment by obdulia-losantos** [comment]
Hopefully this time works :crossed_fingers:

**Comment by admbtlr** [comment]
And now nginx never starts up 💢

**Review by admbtlr — APPROVED** [review]
Everything works fine on a more powerful machine 🎉

**Comment by obdulia-losantos** [comment]
:tada:

**Pull request merged**